src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Ball {
+    String ballColor;
+    int ballNumber;
+
+    public Ball() {
```

Comment:
Unused Constructor. The default no-argument constructor public Ball() {} is not being used. If it's not necessary, you can consider removing it to avoid confusion and potential misuse.